src/sage/rings/polynomial/symmetric_ideal.py

```diff
@@ -23,20 +23,21 @@
 
 Note that ``I`` is not a symmetric Groebner basis::
 
-    sage: G = R*I.groebner_basis()
-    sage: G
-    Symmetric Ideal (x_1^2 + x_1, x_2 - x_1) of Infinite polynomial ring in x over Rational Field
-    sage: Q = R.quotient(G)
-    sage: p = x[3]*x[1]+x[2]^2+3
-    sage: Q(p)
+    sage: G = R * I.groebner_basis()                                                    # optional - sage.combinat
+    sage: G                                                                             # optional - sage.combinat
+    Symmetric Ideal (x_1^2 + x_1, x_2 - x_1) of
+     Infinite polynomial ring in x over Rational Field
+    sage: Q = R.quotient(G)                                                             # optional - sage.combinat
+    sage: p = x[3]*x[1] + x[2]^2 + 3                                                    # optional - sage.combinat
+    sage: Q(p)                                                                          # optional - sage.combinat
     -2*x_1 + 3
```

Comment:
No `singular`?

Reply:
Right now, a missing `sage.combinat` is causing the first error -- for a bad reason, namely permutations. I'll check whether I can include those in **sagemath-categories** already; or alternatively, to ship infinite polynomials instead alongside `sage.combinat` or `sage.groups`.